rare/components/tabs/games/cloud_save_utils.py

```diff
@@ -265,8 +267,8 @@ def game_finished(self, app_name, ignore_settings=False, always_ask: bool = Fals
                 None,
                 "No saves",
                 self.tr(
-                    "There are no saves local and online. Maybe you have to change save path of {}"
-                ).format(igame.title),
+                    f"There are no saves local and online. Maybe you have to change save path of {igame.title}"
```

Comment:
Here's another one where formatting afterwards gets replaced by formatting before translation, I'll leave others unmarked for now since I don't know if I'm even right.